test/unit/api/threads.spec.ts

```diff
@@ -238,6 +240,50 @@ describe('Thread Tests', () => {
     });
   });
 
+  describe('/bulkThreads', () => {
+    it('should pass with public chain', async () => {
```

Comment:
same here wit description, more clarity on what this is testing is the move.

Reply:
"should return bulk threads for a public chain"